### PR TITLE
[WR-363] Update Register Identity documentation

### PR DESCRIPTION
**Pull request by Mrhea**
Spent a moment doing discovery into the SDKs. Here are updated docs for registering an Identity.

### CODE REVIEW

**Comment by pirtleshell** [comment]
sorry this isn't clear, but you should update the JSDoc comments and regenerate docs with `npm run docs`. the main readme contains some basic usage examples that should probably be links to the explanations in the `docs` folder

**Comment by Mrhea** [comment]
> sorry this isn't clear, but you should update the JSDoc comments and regenerate docs with `npm run docs`. the main readme contains some basic usage examples that should probably be links to the explanations in the `docs` folder

I think it did it automatically, I just forgot to commit it! Check now, I believe the correct doc/README.md is there

**Comment by pirtleshell** [comment]
i think you should add your note about this being internal [here](https://github.com/tozny/js-account-sdk/blob/master/src/client.ts#L944) so it shows up in [these generated docs](https://github.com/tozny/js-account-sdk/blob/master/doc/classes/Client.md#registeridentity)

**Review by pirtleshell — APPROVED** [review]

**Review by efabens — APPROVED** [review]

File: src/client.ts [hunk 1]
```diff
@@ -943,6 +943,8 @@ class Client {
    *
    * Note that the `identity` input takes snake_case values.
    *
+   * registerIdentity() is intended for internal use. The JS SDK (@toznysecure/sdk/node) should be used instead to register Identities.
+   *
```
Comment (galxy25) [inline src/client.ts:947]: please add: what are the limitations when using this method, and a brief explainer on why (e.g. no notes are written for the broker to be able to do password recovery/reset)

**Review by galxy25 — CHANGES_REQUESTED** [review]
see comments, and can you update this dependency as part of the release 
https://github.com/tozny/js-account-sdk/security/dependabot/package-lock.json/node-fetch/open

File: package-lock.json [hunk 2]
```diff
@@ -6912,11 +6912,41 @@
       }
     },
     "node_modules/node-fetch": {
-      "version": "2.6.1",
-      "resolved": "https://registry.npmjs.org/node-fetch/-/node-fetch-2.6.1.tgz",
-      "integrity": "sha512-V4aYg89jEoVRxRb2fJdAg8FHvI7cEyYdVAh94HH0UIK8oJxUfkjlDQN9RbMx+bEjP7+ggMiFRprSti032Oipxw==",
+      "version": "2.6.7",
```
Comment (Mrhea) [inline package-lock.json:6915]: Updated with `npm audit` and `npm audit fix`

File: doc/README.md [hunk 3]
```diff
@@ -360,19 +361,30 @@ while (!idList.done) {
...
-  first_name: 'firstName',
-  last_name: 'lastName',
-}
-// Register Identity
-const registeredIdentity = await accountClient.registerIdentity(
+
+// Create a Realm object
+const realmName = 'realmName'
+const appName = 'account'
+const brokerTargetUrl = 'http://platform.local.tozny.com:8000'
```
Comment (galxy25) [inline doc/README.md:372]: this isn't the right format for this, see https://github.com/tozny/flutter_plugin#end-to-end-tozid-identity-registration-login--tozstore-encrypted-filed-storage-example for an example of what this looks like.

File: doc/README.md [hunk 4]
```diff
@@ -360,19 +361,30 @@ while (!idList.done) {
...
+const apiUrl = 'http://platform.local.tozny.com:8000'
```
Comment (galxy25) [inline doc/README.md:373]: think of the audience, they do know about platform.local.tozny or about api.e3db.com?

**Review by galxy25 — CHANGES_REQUESTED** [review]
almost there, see new comments

**Review by galxy25 — APPROVED** [review]

**Pull request merged**